src/mpid/ch4/src/ch4_types.h

```diff
@@ -322,12 +322,17 @@ extern MPL_dbg_class MPIDI_CH4_DBG_MAP;
 extern MPL_dbg_class MPIDI_CH4_DBG_COMM;
 extern MPL_dbg_class MPIDI_CH4_DBG_MEMORY;
 #endif
+
 #define MPIDIU_THREAD_PROGRESS_MUTEX  MPIDI_global.m[0]
 #define MPIDIU_THREAD_PROGRESS_HOOK_MUTEX  MPIDI_global.m[1]
 #define MPIDIU_THREAD_UTIL_MUTEX  MPIDI_global.m[2]
 /* Protects MPIDIG global structures (e.g. global unexpected message queue) */
 #define MPIDIU_THREAD_MPIDIG_GLOBAL_MUTEX  MPIDI_global.m[3]
 #define MPIDIU_THREAD_SCHED_LIST_MUTEX  MPIDI_global.m[4]
-#define MPIDIU_THREAD_TSP_QUEUE_MUTEX  MPIDI_global.m[3]
+#define MPIDIU_THREAD_TSP_QUEUE_MUTEX  MPIDI_global.m[5]
```

Comment:
With the fix in the previous commit, the diff on this line would go away.

Reply:
resolved with new commit